src/db/migration/migration_map.rs

```diff
@@ -470,3 +471,7 @@ mod tests {
         assert!(migration.post_checks(temp_dir.path()).is_ok());
     }
 }
+
+pub(crate) fn db_name(from: &Version, to: &Version) -> String {
```

Comment:
Do we still need it in the trait? Let's simplify things and just have a free `temporary_db_name` function. I guess it _may_ be overridden in the future, but probably YAGNI.

Reply:
It is not in the trait.

Reply:
Let me rephrase. `temporary_db_name` _is_ in the trait, and it's just a proxy for `db_name`. We could use the `db_name` directly (though renamed to be explicit for the purpose). Less code, less bugs. What do you think?

Reply:
Yeah, that makes sense. I'll incorporate that.

Reply:
I just realised that `temporary_db_name` is never used outside of any given migration, so there is no reason to have it as a trait member in the first place. This has been addressed.